src/components/fieldset/address-wrapper.scss

```diff
@@ -0,0 +1,7 @@
+@import "@govuk-frontend/globals/common";
+
+.govuk-c-fieldset {
```

Comment:
I'd suggest adding a comment here to explain what the intent is - basically just add what you wrote in the PR description!

Reply:
Done